packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -0,0 +1,34 @@
+
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Adds Navigation links', () => {
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	it( 'Should add a link with one click', async () => {
+		const navigationAppender = '.wp-block-navigation .block-list-appender';
+		const emptyOption = '.wp-block-navigation-placeholder__button.is-link';
+		const linkInput = '.block-editor-link-control__search-input';
+		const navigationLink = '.wp-block-navigation-link';
```

Comment:
Not sure that putting these into constants is helpful—I keep finding myself having to scroll up to figure out what's going on 😀